src/extension/src/file_handlers/JsonFileHandler.py

```diff
@@ -74,7 +72,7 @@ def write_to_json_file(self, dir_path, file, content):
                         return
                 except Exception as error:
                     error_message = "Trial {0}: Could not write to file. [File={1}] [Location={2}] [Exception={3}]".format(retry+1, file, str(file_path), error)
-                    self.logger.log_error(error_message)
+
```

Comment:
Same thing here.

Reply:
Same reason as mentioned in the other comment